lib/shopify_api/rest/base.rb

```diff
@@ -116,11 +116,17 @@ def has_many?(attribute)
           @has_many.include?(attribute)
         end
 
+        sig { returns(T::Hash[Symbol, Class]) }
+        attr_reader :has_many
```

Comment:
I wonder if it makes sense to declare all of the readers around L56 above so they're more grouped up?